docs/metrics.md

```diff
@@ -46,7 +46,8 @@ bucketlistDB-live.bulk.poolshareTrustlines  | timer     | time to load poolshare
 bucketlistDB-live.bulk.prefetch             | timer     | time to prefetch
 bucketlistDB-live.bulk.eviction           | timer     | time to load for eviction scan
 bucketlistDB-live.bulk.query              | timer     | time to load for query server
-bucketlistDB-<X>.point.<Y>                | timer     | time to load single entry of type <Y> on BucketList <X> (live/hotArchive)
+bucketlistDB-<X>.point-time.<Y>           | counter   | sum of time (microseconds) to load single entry of type <Y> on BucketList <X> (live/hotArchive)
```

Comment:
what do you think about `bucketlistDB-<X>.point-time.<Y>` --> `bucketlistDB-<X>.<Y>.sum`? (and similar with count)